docs/extending/extending_form_builder.md

```diff
@@ -215,6 +215,17 @@ By adjusting them to your needs, you will change your email template.
 
 ### Configure sender details
 
-To send emails, you also need to configure `sender_address` in `config/packages/swiftmailer.yaml`.
-It acts as a sender and return address for all bounced messages.
-For details, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/5.0/reference/configuration/swiftmailer.html#sender-address)
+Some e-mail providers require the `sender_address` in `config/packages/swiftmailer.yaml` to be set. Therefore, to send notification emails from the Form Builder, you need to configure it . It acts as a sender and return address for all bounced messages. For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/5.0/reference/configuration/swiftmailer.html#sender-address) Some providers require a `sender_address` to be defined, or outgoing mails will be bounced.
```

Comment:
```suggestion
Some email providers require a sender address to be set, so to avoid unsent emails when using Form Builder,
it is recommended to configure `sender_address` in `config/packages/swiftmailer.yaml`.
This email acts as a sender and return address for all bounced messages.
For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/5.0/reference/configuration/swiftmailer.html#sender-address)
```
if all the facts are ok, to mirror the structure from the other Pr